src/MOI_wrapper.jl

```diff
@@ -152,7 +181,7 @@ function MOI.get(o::Optimizer, ::MOI.TimeLimitSec)
 end
 
 function MOI.set(o::Optimizer, ::MOI.TimeLimitSec, value)
-    if value === nothing
+    if value == nothing
```

Comment:
why did you change this? `===` should be preferred when comparing with nothing

Reply:
I thought I made a typo here, have restored the original === now